session.py

```diff
@@ -12,15 +12,16 @@ def __init__(self):
         client = self.BackendApplicationClient(client_id = self.CLIENT_ID)
         self.oauth2_session = self.OAuth2Session(client = client)
         token = self.oauth2_session.fetch_token(
-                config['auth-endpoint'],
-                client_id = self.CLIENT_ID,
-                client_secret = config['secret'],
-                scope = 'auto')
+            config['auth-endpoint'],
+            client_id = self.CLIENT_ID,
+            client_secret = config['secret'],
+            scope = 'auto')
         self.account = int(config['account'] or token['Permissions'][0]['AccountId'])
 
     # request synchronously communicates with the Wild Apricot API
-    def request(self, verb, endpoint, params = {}, data = {}):
-        endpoint = f'/v2.1/accounts/{self.account}/{endpoint}'
+    def request(self, verb, endpoint, params = {}, data = {}, rpc = False):
+        path_prefix = "accounts" if not rpc else "rpc"
+        endpoint = f'/v2.1/{path_prefix}/{self.account}/{endpoint}'
```

Comment:
I might write this as `f'/v2.1/{'rpc' if rpc else 'accounts'}/{self.account}/{endpoint}'`, to save a line.